msu/utils/serialization.nut

```diff
@@ -16,6 +16,18 @@
 		"Array",
 		"SerializationData"
 	]),
+	IsSaving = false, // Is flipped during world_state.saveCampaign
+	IsLoading = false, // Is flipped during world_state.loadCampaign
+
+	function isSaving()
+	{
+		return this.IsDuringSerialization;
+	}
+
+	function isLoading()
+	{
+		return this.IsDuringDeserialization;
+	}
```

Comment:
Should this be added here or in `world_state`?

Reply:
in `serialization.nut` the name `isLoading` is less ambiguous.

One might also consider the complete loading-screen between combat and world map to be "loading", even though there is no serialization happening there, other than a short autosave

Reply:
I was hoping to have shorter names than `IsDuringSerialization`. Is this setup clear enough with `::MSU.Serialization.isSaving()` and `::MSU.Serialization.isLoading()`? Or is it important to use the `serializing` and `deserializing` nomenclature?

Reply:
With it being in the Serialization namespace, I think that should be rather clear.